tests/storage-disks-vm

```diff
@@ -110,9 +110,23 @@ lxc exec v1 -- ls -l /mnt/foo1
 lxc stop -f v1
 
 # Check adding a disk to a vm with a long name (max 63) and slash to test possible problems with long socket paths or long qemu device tags
-LONG_DEVICE_NAME="device-with-very-long-name-and-/-4-qemu-property-handling-test_"
+LONG_DEVICE_NAME="device-/-with-very-long-name-and-4-qemu-property-handling-test_"
```

Comment:
What was the reason for the / change here?

Reply:
The reason was to make `/` show on the `/dev/disk/by-id` path so we can check for escaping